src/csvcubed/utils/sparql_handler/sparql_queries/select_codelist_cols_by_dataset_url.sparql

```diff
@@ -4,20 +4,25 @@ PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
 PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
 
 
-SELECT DISTINCT ?columnPropertyUrl ?columnValueUrl ?columnTitle
+SELECT DISTINCT ?columnPropertyUrl ?columnValueUrl ?columnTitle ?columnName
 WHERE {   
 
     ?table csvw:url ?tableUrl.
     ?table csvw:tableSchema/csvw:column/rdf:rest*/rdf:first ?column.
 
-    ?column csvw:propertyUrl ?columnPropertyUrl.
+    OPTIONAL{
```

Comment:
So this change is in here because we want to bring out the unique identifier column which is suppressed and doesn't have a property URL?

Reply:
@robons Correct